docs/CHANGELOG.asciidoc

```diff
@@ -77,6 +78,353 @@ Review important information about the {kib} 8.x releases.
+Deprecated functionality does not have an immediate impact on your application, but we strongly recommend
+you make the necessary updates after you upgrade to 8.16.0.
+
+[discrete]
+* The Logs Stream is now hidden by default in favor of the Logs Explorer app.
+[%collapsible]
+====
+*Details* +
+You can find the Logs Explorer app in the navigation menu under Logs > Explorer, or as a separate tab in Discover. For more information, refer to ({kibana-pull}194519[#194519]).
+
+*Impact* +
+You can still show the Logs Stream app again by navigating to Stack Management > Advanced Settings and by enabling the `observability:enableLogsStream` setting.
+====
+      
+[discrete]
+* Deprecates the Observability AI Assistant specific advanced setting `observability:aiAssistantLogsIndexPattern`.
+[%collapsible]
+====
+*Details* +
+The Observability AI Assistant specific advanced setting for Logs index patterns `observability:aiAssistantLogsIndexPattern` is deprecated and no longer used. The AI Assistant will now use the existing **Log sources** setting `observability:logSources` instead. For more information, refer to ({kibana-pull}192003[#192003]).
+
+//*Impact* +
+//!!TODO!!
+====
+    
+
+      
+[float]
+[[features-8.16.0]]
+=== Features
+{kib} 8.16.0 adds the following new and notable features.
+
+AGPL license::
+* Adds AGPL 3.0 license ({kibana-pull}192025[#192025]).
+Alerting::
+* Support TheHive connector to cases ({kibana-pull}180931[#180931]).
+* TheHive Case Connector ({kibana-pull}180138[#180138]).
+* Inference Service Kibana connector ({kibana-pull}189027[#189027]).
+Dashboards and visualizations::
+* Adds the ability to star your favorite dashboards and quickly find them ({kibana-pull}189285[#189285]).
+* Adds a chart showing usage statistics to the dashboard details ({kibana-pull}187993[#187993]).
+* Adds support for coloring table cells by terms with color mappings assignments. This is supported for both Rows and Metric dimensions ({kibana-pull}189895[#189895]).
+Data ingestion and Fleet::
+* Support content packages in UI ({kibana-pull}195831[#195831]).
+* Advanced agent monitoring options UI for HTTP endpoint and diagnostics ({kibana-pull}193361[#193361]).
+* Adds option to have Kafka dynamic topics in outputs ({kibana-pull}192720[#192720]).
+* Adds support for GeoIP processor databases in Ingest Pipelines ({kibana-pull}190830[#190830]).
+////
+!!TODO!! The above PR had a lengthy release note description:
+The Ingest Pipelines app now supports adding and managing databases for the GeoIP processor. Additionally, the pipeline creation flow now includes support for the IP Location processor.
+////
+* Adds agentless ux creation flow ({kibana-pull}189932[#189932]).
+* Enable feature flag for reusable integration policies ({kibana-pull}187153[#187153]).
+Discover::
+* When editing ES|QL queries, you now get recommendations to help you get started ({kibana-pull}194418[#194418]).
```

Comment:
Editing seems wrong here, recommended queries appear every time you typing an ES|QL query not only when you are editing 🤔

Reply:
```suggestion
* When writing ES|QL queries, you now get recommendations to help you get started ({kibana-pull}194418[#194418]).
```
That's what it meant but I agree it could be confusing ^^'